library/oneview_server_profile_template.py

```diff
@@ -193,8 +194,9 @@ def __create(self, data):
 
     def __update(self, data):
         merged_data = ServerProfileMerger().merge_data(self.current_resource.data, data)
-
-        equal = compare(merged_data, self.current_resource.data)
+        updated_data = deepcopy(merged_data)
+        updated_data.pop('initialScopeUris', None)
```

Comment:
before the compare why are you removing InitialScopeUris from updated_data dict?

Reply:
As per the API docs, initialScopeUris are provided only at creation time and cannot be displayed in the GET response of the created resource.

So if a user executes the same playbook again, then the mismatch happens between GET response and provide yaml and it says the resource is updated (which is not actually updated). So to achieve idempotency checks, I have deleted the initialScopeUris key in update method.

Reply:
so in that case, what if customer wants to do a update operation for ScopeURIs to SP? 
this will not work.

Reply:
there is another key call "scopesUri". We have to use that key for such operation.